app/src/main/AndroidManifest.xml

```diff
@@ -16,31 +16,30 @@
         tools:targetApi="31">
         <activity
             android:name=".SymptomsActivity"
-            android:exported="false" />
+            android:exported="false"
+            android:screenOrientation="portrait"/>
         <activity
             android:name=".Health"
-            android:exported="false" />
+            android:exported="false"
+            android:screenOrientation="portrait"/>
         <activity
             android:name=".Home"
-            android:exported="true">
+            android:exported="true"
+            android:screenOrientation="portrait">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-
         <activity
             android:name=".Today"
-            android:exported="false"></activity>
-
-        <receiver android:name=".NotificationAlarmManager" />
```

Comment:
Should this line be removed: <receiver android:name=".NotificationAlarmManager" />